config/default.yml

```diff
@@ -503,6 +503,16 @@ RSpec/ImplicitSubject:
   VersionChanged: '2.13'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ImplicitSubject
 
+RSpec/IndexedLet:
+  Description: Do not set up test data using indexes (e.g., `item_1`, `item_2`).
+  Enabled: pending
+  VersionAdded: "<<next>>"
+  Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/IndexedLet
+  MaxRepeats: 3
```

Comment:
Personally, I'd vote for no indexed definitions. If some would like to tolerate 3 indexed definitions, let's keep this configurable, but I would set the default to 1.

Reply:
agreed, now it's one